buildscripts/ci/backend/docker/setup.sh

```diff
@@ -105,7 +105,13 @@ echo "Installing Google Fonts..."
 mkdir -p "$FONTS_DIR"
 find . -type f \( -iname "*.ttf" -o -iname "*.otf" \) -print0 | xargs -0 -r mv -t "$FONTS_DIR"
 
-echo "Installing Fonts Cache..."
+echo "Installing MS Fonts..."
```

Comment:
Initially I read the below "mscorefonts" as "mscore fonts", but it really is MS Core Fonts, so this text above should better read "Installing MS Core Fonts" or even "Installing Microsoft Core Fonts"